.github/workflows/release_dependency_versions.yml

```diff
@@ -7,20 +7,17 @@ on:
 env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-permissions:  # added using https://github.com/step-security/secure-workflows
-  contents: read
```

Comment:
any particular reason to remove this?

Reply:
See #37028. Seems to be a permission error when triggering the workflow on the other repo, seemingly because `GITHUB_TOKEN` doesn't have the right permissions.  Maybe `actions: write` would do it, but I don't see any clear documentation around it.